relay-general/src/store/mod.rs

```diff
@@ -137,3 +140,25 @@ impl<'a> Processor for StoreProcessor<'a> {
         Ok(())
     }
 }
+
+pub fn light_normalize<'a>(
+    event: &mut Annotated<Event>,
+    client_ip: Option<&IpAddr>,
+    user_agent: Option<&'a str>,
+    received_at: Option<DateTime<Utc>>,
+    max_secs_in_past: Option<i64>,
+    max_secs_in_future: Option<i64>,
+    breakdowns_config: Option<BreakdownsConfig>,
+) -> ProcessingResult {
+    validate_annotated_transaction(event)?;
+    light_normalize_event(
```

Comment:
nit: If my search is correct, `light_normalize_event` is never called outside of `light_normalize`. So why not simplify this by calling `validate_annotated_transaction` from inside `light_normalize_event` (and then remove the `light_normalize` wrapper)?

Reply:
Good feedback, I'll do that in a follow-up PR.

Reply:
+1 on this, would even prefer if this goes into the same PR.

Reply:
Addressed in https://github.com/getsentry/relay/pull/1385.